common/config/BUILD

```diff
@@ -0,0 +1,17 @@
+java_library(
+    name = "config",
+    srcs = ["Config.java", "ConfigKey.java", "SystemProperty.java"],
```

Comment:
Is there a reason why we don't use a `glob([**/*.java])` here, @vmax ?

Reply:
@haikalpribadi apart from explicit file lists being a bit faster than recursive `glob`, there is none